### ISSUE TITLE
content task - set tag synonyms

### ISSUE THREAD

**Issue by OriHoch** [hasadna/Open-Knesset#364]
we have many duplicate tags with different spellings, for example:
* https://oknesset.org/tags/%D7%91%D7%99%D7%98%D7%97%D7%95%D7%9F/
* https://oknesset.org/tags/%D7%91%D7%98%D7%97%D7%95%D7%9F/

Need to go over tags admin and set synonyms

#### Pre-requisites
* ask an admin for access to django admin - tags & tags_synonyms tables
* ask an admin for tagging.add_tag permission

#### Steps
0. login to django admin > tags table
0. on the left - filter by "parent tags"
0. look for duplicated tags
0. click on "הוספת תג נרדף" next to the parent tag - the correct tag
0. now, look for the synonym and click on mark as synonym next to it
0. you can mark more synonyms / look for synonyms on other pages as well
0. when you are done click on "cancel"
0. repeat

**Comment by morchickit** [comment]
@OriHoch - is this still needed to be done manually? If so, I'll add it to the doc...

**Comment by OriHoch** [comment]
yes, still needs to be done, I think @kanija did some work on it

**Comment by morchickit** [comment]
@OriHoch - Do we want to add it as a hackathon task?

**Comment by OriHoch** [comment]
yes, sounds like a good task for the hackathon
